Pod/Classes/SwiftPhotoGallery.swift

```diff
@@ -28,9 +28,16 @@ public class SwiftPhotoGallery: UIViewController, UICollectionViewDataSource, UI
     public var imageCollectionView: UICollectionView!
     public var numberOfImages: Int = 0
 
+    public var backgroundColor: UIColor = UIColor.blackColor()
+
     public var currentPage: Int {
         set(page) {
-            imageCollectionView.scrollToItemAtIndexPath(NSIndexPath(forItem: page, inSection: 0), atScrollPosition: .CenteredHorizontally, animated: false)
+            if page <= numberOfImages - 1 {
```

Comment:
Since `numberOfImages` is an `Int`, couldn't you simplify this as `if page < numberOfImages`?